apps/condo/domains/subscription/hooks/useFeatureSubscription.tsx

```
@@ -1,3 +1,5 @@
+import getConfig from 'next/config'
```

ESLint

[error] There should be no empty line within import group

(import/order)